packages/search-ui/src/types/index.ts

```diff
@@ -110,7 +110,7 @@ export type SuggestionsQueryConfig = {
 export type FacetConfiguration = {
   type: string;
   size?: number;
-  ranges?: FilterValueRange | FilterValueRange[];
```

Comment:
I cant find an example of this, not sure if this was an accident in the types. Documentation on site-search says that it must be an array. Couldn't find documentation on elastic's app-search API.

https://swiftype.com/documentation/app-search/api/search/facets#range-facet

Reply:
Is this the one you were looking for? https://www.elastic.co/guide/en/app-search/current/facets.html#facets-range-facet